doc/content/source/materials/ConcreteExpansionMicrocrackingDamage.md

```diff
@@ -0,0 +1,76 @@
+# Concrete Expansion Microcracking Damage
+
+!syntax description /Materials/ConcreteExpansionMicrocrackingDamage
+
+## Description
+
+The `ConcreteExpansionMicrocrackingDamage` model computes a scalar damage index $d$ that increases with the progress of the eigenstrain defined by the `eigenstrain_name` input parameter (for example, the eigenstrain induced by alkali-silica reaction (ASR) using a [ConcreteASREigenstrain](ConcreteASREigenstrain.md) model) and the current stress state of the material.
```

Comment:
Close the left parentheses '(for example...'

Reply:
It is closed after 'model) ...'